third-party/jemalloc/Makefile

```diff
@@ -15,7 +15,8 @@ CHPL_JEMALLOC_CFG_OPTIONS += --host=$(CHPL_MAKE_TARGET_PLATFORM)-unknown-linux-g
 endif
 
 CHPL_JEMALLOC_CFG_OPTIONS += --prefix=$(JEMALLOC_INSTALL_DIR) \
-			     --with-jemalloc-prefix=je_
+                             --with-jemalloc-prefix=je_ \
+                             --with-private-namespace=jeint_
```

Comment:
I put an example of how we might do this in #5566 .  If you like it, you can approve that PR, or we can discuss doing something else.